tests/Offer/OfferTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\CalendarSummaryV3\Offer;
+
+use DateTimeImmutable;
+use PHPStan\Testing\TestCase;
+
+final class OfferTest extends TestCase
```

Comment:
Do we also check the smaller methods just to be safe(r)?

Reply:
Aren't they all covered by the integration tests?